### DISCUSSION TITLE
Is micronaut framework (micronaut-security) FIPS 140-2 complaint ?

### DISCUSSION THREAD

**Discussion by Anand-Kotamsetti** [micronaut-projects/micronaut-security#1584] [Q&A]
We use micronaut framework for a micro service and would like to know whether  micronaut framework (micronaut-security) FIPS 140-2 complaint ? 

Thanks in advance 👍

**Comment by sdelamo** [comment]
We have not tested Micronaut security against [FIPS 104-2](https://csrc.nist.gov/pubs/fips/140-2/upd2/final) or [FIPS 140-3](https://csrc.nist.gov/pubs/fips/140-3/final).
